src/nested_taylor.jl

```diff
@@ -0,0 +1,56 @@
+# This file is part of the TaylorSeries.jl Julia package, MIT license
+#
+# Luis Benet & David P. Sanders
+# UNAM
+#
+# MIT Expat license
+#
+
+
+#number of variables in a nested polynomial
+function get_nested_numvars{T<:Number}(a::Taylor1{T})
```

Comment:
There may be some type instability in this function, since `a_aux` may change of type. I suggest to do the same but recursively (so call again `get_nested_numvars`), which may require to define another method, where `a` is not an `AbstractSeries`.